library/oneview_uplink_set.py

```diff
@@ -93,13 +100,14 @@
     type: complex
 '''
 
+UPLINK_SET_KEY_REQUIRED = "Uplink Set Name and Logical Interconnect required."
 UPLINK_SET_CREATED = 'Uplink Set created successfully.'
 UPLINK_SET_UPDATED = 'Uplink Set updated successfully.'
 UPLINK_SET_DELETED = 'Uplink Set deleted successfully.'
 UPLINK_SET_ALREADY_EXIST = 'Uplink Set already exists.'
 UPLINK_SET_ALREADY_ABSENT = 'Nothing to do.'
-UPLINK_SET_NOT_EXIST = 'Rename failed: Uplink Set not found.'
```

Comment:
@tmiotto 

I found an issue on this module and I fixed it. I forgot to add an observation about that on the PR.

Before the fix, when set a newName in the data on the 'present' state, the update method was always being called, changing the name of the Uplink Set, and ignoring the rest of the data provided on the play.

For the 'present' state, the expected behavior is create a resource if it doesn't exist. Otherwise, it updates the resource with the changes provided, including the name (when a newName is set). 

Therefore, that message 'Rename failed: Uplink Set not found.', specific for the rename, doesn't make sense anymore.